media/css/firefox/firstrun/firstrun_quantum.less

```diff
@@ -0,0 +1,405 @@
+@import "../../sandstone/lib.less";
+
+@font-face {
+    font-family: 'Fira Sans Light';
+    src: url('/media/fonts/FiraSans-Light.woff2') format('woff2'),
+         url('/media/fonts/FiraSans-Light.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'Fira Sans Light';
+    src: url('/media/fonts/FiraSans-SemiBold.woff2') format('woff2'),
+         url('/media/fonts/FiraSans-SemiBold.woff') format('woff');
+    font-weight: bold;
+    font-style: normal;
+}
+
+:root {
+    --animation-curve: ease-in;
+}
+
+body {
+    margin: 0;
+}
+
+* {
+    font-family: 'Fira Sans Light';
+}
+
+#browser-wrapper {
+    min-height: 100%;
+    min-width: 100%;
+    position: fixed;
+    display: block;
+    overflow: auto;
+    width: 100%;
+}
+
+ img {
+    bottom: 0;
+    height: auto;
+    left: 0;
+    margin: auto;
+    min-height: 100%;
+    min-width: 100%;
+    object-fit: cover;
+    position: fixed;
+    right: 0;
+    top: 0;
+    width: 100%;
+}
+
+#firefox-logo {
+    background: url('/media/img/firefox/firstrun-57/firefox_and_wordmark.png');
```

Comment:
This URL should be `img/firefox/firstrun-quantum/firefox_and_wordmark.png` to fix the build error